projects/kubernetes/kubernetes/Makefile

```diff
@@ -86,11 +86,13 @@ docker: binaries pause
 		--build-arg BASE_IMAGE=$(GO_RUNNER_IMAGE) \
 		--build-arg VERSION=$(IMAGE_TAG) \
 		-t $(PAUSE_IMAGE) \
+		-t $(IMAGE_REPO)/$(IMAGE_REPO_PREFIX)/$(PAUSE_IMAGE_NAME):3.2 \
```

Comment:
Any idea how often the version of pause upstream changes?  Is 3.2 something that we need to be looking out to change when updating kub versions?

Reply:
1.20 k8s still uses 3.2 pause. We won't be changing it often.

Reply:
I'm going to look at making kubeadm configurable for pause version, but that won't be available (if I get something through) until 1.21.

Reply:
sgtm